k2/csrc/host/arcsort_test.cc

```diff
@@ -1,21 +1,26 @@
-// k2/csrc/arcsort_test.cc
-
-// Copyright (c)  2020  Xiaomi Corporation (author: Haowen Qiu)
-
-// See ../../LICENSE for clarification regarding multiple authors
-
-#include "k2/csrc/arcsort.h"
+/**
+ * @brief
+ * arcsort_test
+ *
+ * @copyright
+ * Copyright (c)  2020  Xiaomi Corporation (authors: Haowen Qiu)
+ *
+ * @copyright
+ * See LICENSE for clarification regarding multiple authors
+ */
+
+#include "k2/csrc/host/arcsort.h"
 
 #include <algorithm>
+#include <gmock/gmock.h>
```

Comment:
Please use `""` for third party libraries.

`<>` is reserved for standard headers.

Please follow the google c++ style for ordering headers:
<https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes>

> 1. dir2/foo2.h.
> 2. A blank line
> 3. C system headers (more precisely: headers in angle brackets with the .h extension), e.g., <unistd.h>, <stdlib.h>.
>4. A blank line
>5. C++ standard library headers (without file extension), e.g., <algorithm>, <cstddef>.
>6. A blank line
>7. Other libraries' .h files.
>8. Your project's .h files.

Reply:
About the "", could you give where is the google style requirements?

Reply:
Not sure whether there are any problems with your browser. I have already posted the URL in the comment.

Reply:
and about the order, you are right, but I was confused about it cause the old codes did not follow this. Though may you not review all codes, I want to change it in eigher situation. But to avoid this style things keep bother us, I suggest a robot be setup for this. It will save man time and not require hard review.

Reply:
> I choose a revert to host

Anyway, make sure you code get compiled and pass the tests, we can fix `host` issue later.

Reply:
I dont have strong opinion RE angle brackets or quotes, fangjun may be right but not so important so you can forget it.